dashboard/test/ui/features/javalab/code_review_eyes.feature

```diff
@@ -1,7 +1,6 @@
 @eyes
 @no_mobile
 @no_ie
-@skip
```

Comment:
nit: since you're in these files can you  also remove the @no_ie tag? I think we can safely remove it now that we no longer support ie.

Reply:
Can do!